tools/api-linter/src/main.rs

```diff
@@ -64,32 +133,61 @@ fn main() -> Result<()> {
         Default::default()
     };
 
-    println!(
+    eprintln!(
         "This build understands rustdoc format version {}",
         FORMAT_VERSION
     );
-    println!("Running rustdoc to produce json doc output...");
-    let package = cargo::CargoRustDocJson::new(&args.crate_path, args.target_path)
-        .run()
-        .context(here!())?;
+    eprintln!("Running rustdoc to produce json doc output...");
+    let package = cargo::CargoRustDocJson::new(
+        &args.crate_path,
+        &args.target_path,
+        args.nightly_version.take(),
+        args.features(),
+    )
+    .run()
+    .context(here!())?;
 
-    println!("Examining all public types...");
+    eprintln!("Examining all public types...");
     let errors = Visitor::new(config, package)?.visit_all()?;
-    let mut error_printer = ErrorPrinter::new(&args.crate_path);
-    for error in &errors {
-        println!("{}", error);
-        if let Some(location) = error.location() {
-            error_printer
-                .pretty_print_error_context(location, error.subtext())
-                .context("failed to output error context")?;
+    match args.output_format {
+        OutputFormat::Errors => {
+            let mut error_printer = ErrorPrinter::new(&args.crate_path);
+            for error in &errors {
+                println!("{}", error);
+                if let Some(location) = error.location() {
+                    error_printer
+                        .pretty_print_error_context(location, error.subtext())
+                        .context("failed to output error context")?;
+                }
+            }
+            if !errors.is_empty() {
+                println!(
+                    "{} {} emitted",
+                    errors.len(),
+                    "errors".if_supports_color(Stream::Stdout, |text| text.red())
+                );
+            }
+        }
+        OutputFormat::MarkdownTable => {
+            println!("| Crate | Type | Used In |");
+            println!("| ---   | ---  | ---     |");
```

Comment:
if we want this to actually be human readable, seems like we should compute the width of these columns ? non-blocking though obviously this isn't critical